test/AuthSamples.FunctionalTests/CookiesTests.cs

```diff
@@ -37,8 +36,59 @@ public CookiesTests(WebApplicationFactory<Cookies.Startup> fixture)
             var content = await response.Content.ReadAsStringAsync();
 
             // Assert
-            Assert.Equal(HttpStatusCode.Redirect, response.StatusCode);
-            Assert.Equal("http://localhost/account/login?ReturnUrl=%2FHome%2FMyClaims", response.Headers.Location.ToString());
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Equal("http://localhost/account/login?ReturnUrl=%2FHome%2FMyClaims", response.RequestMessage.RequestUri.ToString());
+        }
+
+        [Fact]
+        public async Task MyClaimsShowsClaimsWhenLoggedIn()
+        {
+            // Arrange & Act & Assert
+            await SignIn("Dude");
+            await CheckMyClaims("Dude");
+        }
+
+        [Fact]
+        public async Task LogoutClearsCookie()
```

Comment:
Since these tests run in parallel, does it matter when you are calling Logout as you would be using the same cookie container? (race between logout and checkmyclaims) could changing the name to something else than 'Dude' help?

Reply:
It's parallel per class, there shouldn't be any shared resources.

Reply:
Thanks @Tratcher! I didn't knew that.

Reply:
Yeah these tests definitely would not work running in parallel at the same time, so its a good question :)